frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -118,7 +118,10 @@ public WorkerThread(
 
     @Override
     public void run() {
-        int responseTimeout = model.getResponseTimeout();
+        int responseTimeout =
+                ConfigManager.getInstance().isDebug()
```

Comment:
move this logic into model.getResponseTimeout()

Reply:
Yup. Makes sense.